plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergSystemTables.java

```diff
@@ -639,6 +639,68 @@ void testEntriesTable()
         }
     }
 
+    @Test
```

Comment:
We don't have a test with equality deletes, right? Could you add a test if so? You can refer to helper methods (e.g. writeEqualityDeleteToNationTable) in TestIcebergV2.